actionview/lib/action_view/helpers/form_tag_helper.rb

```diff
@@ -117,17 +117,43 @@ def field_id(object_name, method_name, *suffixes, index: nil, namespace: nil)
         ].tap(&:compact!).join("_")
       end
 
+      # Generate the text-only content that would be used for a the given name and
+      # field combination.
+      #
+      #   field_label :post, :title
+      #   # => "Title"
+      #
+      #   field_label :post, :published, value: true
+      #   # => "Published"
+      #
+      #   field_label :post, :published, value: false
+      #   # => "Draft"
+      #
+      def field_label(object_name, method_name, object: nil, value: nil)
```

Comment:
I'm not entirely sold on this method name, so I would really appreciate some feedback.

I've settled on `field_label` because it mirrors the adjacent [field_id](https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-field_id) and [field_name](https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/FormBuilder.html#method-i-field_name) helpers.

A more verbose option I considered was `field_label_text` to clearly communicate that this is a text-only value. Another (short) alternative I considered was `field_text`, but felt the omission of `label` could miscommunicate the type of value returned.

Reply:
In my opinion, retaining the `field_` prefix would make it feel related to the `field_name` and `field_id` helpers. 

> the simple_form gem uses `label_text` to describe that content

With this in mind (ignoring the potential for naming collisions), is `field_label_text` a better name?

Reply:
> In my opinion, retaining the field_ prefix would make it feel related to the field_name and field_id helpers.

I can see the reasoning behind wanting to keep it as `field_*` however `field_id` and `field_name` are values for attributes on the input element itself whereas this is a separate element but my opinion doesn't matter anymore 😄 

> With this in mind (ignoring the potential for naming collisions), is field_label_text a better name?

It feels cumbersome to me - better to keep it as `field_label`.